docroot/modules/contrib/mirador/js/miradorEndPoint.js

```diff
@@ -218,7 +218,7 @@
         drupalAnnotationStore['text'] = annotation['resource']['0']['chars'];
         drupalAnnotationStore['data'] = {};
         drupalAnnotationStore['data'] = annotation;
-        drupalAnnotationStore['type'] = 'image annotation';
+        drupalAnnotationStore['media'] = 'image annotation';
```

Comment:
@safallia-zyxware Can we keep it just *image* since, the media attribute for videos is just *video*.